api/program-enrollments/models/program-enrollments.settings.json

```diff
@@ -123,6 +123,9 @@
       "plugin": "upload",
       "required": false,
       "pluginOptions": {}
+    },
+    "course_name_other": {
```

Comment:
this looks good.

@Ayush8923 can you also add this field in the students API, `createFromWebhook` controller handler? So that whatever data that we send from the API (Pehel Page), it gets saved in the program enrollment record into this field.

Reply:
Made the changes according to your comment. Please check.